src/components/progress/progress.scss

```diff
@@ -0,0 +1,117 @@
+@import "../../themes/ionic.globals";
+// Progress bar
+// --------------------------------------------------
+
+/// @prop - The color of the element that is being animated
+$progress-color:              rgb(16,108,200) !default;
+
+/// @prop - The background color of the progress bar
+$progress-buffer-color:       rgb(170,209,249) !default;
```

Comment:
Can you please align the value and the value above to the same column as the value below?